environment.yml

```diff
@@ -17,6 +17,9 @@ dependencies:
     - netcdf4
     - dask
     - pip
+    - datetime
```

Comment:
`datetime` is pat of the standard library, you don't have to put it as dependency.